src/lib/utils/typography/Typography.styles.tsx

```diff
@@ -13,30 +13,35 @@ const useTypographyStyle = makeStyles(
       fontSize: '24px',
       fontWeight: 700,
       lineHeight: '28px',
+      marginTop: '28px',
```

Comment:
Same typography concern as in _markdown.scss, seems the line height that design provided is also an appropriate margin (I wonder if that's what was actually intended?).